connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -361,4 +413,67 @@ public Void answer() throws Throwable {
     private  <T> void expectAndCallbackNotLeaderException(final Capture<Callback<T>> cb) {
         expectAndCallbackException(cb, new NotLeaderException("not leader test", LEADER_URL));
     }
+
+    /* Name here needs to be unique as we are testing the aliasing mechanism */
+    public static class WorkerTestConnector extends Connector {
+
+        public static final String TEST_STRING_CONFIG = "test.string.config";
+        public static final String TEST_INT_CONFIG = "test.int.config";
+
+        @Override
+        public String version() {
+            return "1.0";
+        }
+
+        @Override
+        public void start(Map<String, String> props) {
+
+        }
+
+        @Override
+        public Class<? extends Task> taskClass() {
+            return null;
+        }
+
+        @Override
+        public List<Map<String, String>> taskConfigs(int maxTasks) {
+            return null;
+        }
+
+        @Override
+        public void stop() {
+
+        }
+
+        @Override
+        public ConfigDef defineConfig() {
```

Comment:
It doesn't seem like any of the `defineConfig` or `validate` are actually invoked by these tests. Can we stub them out with something a lot simpler?